google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -75,6 +79,8 @@
   private final Map<String, XBreakpoint> myIdeBreakpoints = new ConcurrentHashMap<String, XBreakpoint>();
   private final CloudDebugProcess myProcess;
   private PsiManager myPsiManager;
+  private VirtualFileSystem fileSystem =
```

Comment:
Given that we will need a RepoToLocalFile map (JavaUtil) regardless, I think I will roll back all of this and try to fix my original problem, which is to somehow extract the package name from jamjavatest/guestbook/src/main/java/ninja/joao/guestbook/SignGuestbookServlet.java and get the corresponding VirtualFile from the map.